api/tests/integration/infrastructure/repositories/campaign-participation-repository_test.js

```diff
@@ -953,14 +953,24 @@ describe('Integration | Repository | Campaign Participation', function () {
       describe('Count shared Participation', function () {
         it('counts participations shared', async function () {
           databaseBuilder.factory.buildCampaignParticipation();
-          databaseBuilder.factory.buildCampaignParticipation({ campaignId });
+          databaseBuilder.factory.buildCampaignParticipation({ campaignId, status: SHARED });
           await databaseBuilder.commit();
 
           const result = await campaignParticipationRepository.countParticipationsByStatus(campaignId, campaignType);
 
           expect(result).to.deep.equal({ started: 0, completed: 0, shared: 1 });
         });
 
+        it('should not count shared participations if participation was deleted', async function () {
+          databaseBuilder.factory.buildCampaignParticipation();
```

Comment:
tu peux supprimer cette ligne car elle te permet de vérifier que tu ne récupère que les campaignParticipation d'une campaign précise. Or c'est pas ce qu'on cherche à tester ici et c'est testé dans le it d'avant 😄

Reply:
true.